tests/test_decubitus.py

```diff
@@ -0,0 +1,575 @@
+import unittest
+from pydicom.dataset import Dataset, FileMetaDataset
+from pydicom.sequence import Sequence
+from pydicom.uid import generate_uid, PYDICOM_IMPLEMENTATION_UID
```

Comment:
`dicompyler-core` has a base UID prefix available in `config.py`. It's not necessary to use it as this is just a test file and is not generating any data outside of this test module.

Reply:
You're right, it is not needed at all, I just pulled some boilerplate from some past code I used.  I'll look into just removing it.